lib/msf/base.rb

```diff
@@ -16,22 +16,14 @@
 # framework-base depends on framework-core
 
 # Configuration
-require 'msf/base/config'
-require 'msf/base/logging'
 
 # Simple wrapper
-require 'msf/base/simple'
 
 # Sessions
```

Comment:
I'm guessing these comments can be removed now? 👀

Reply:
I actually have this file being completely removed in it's entirety an upcoming PR, it's no longer necessary but I can clean it up real quick for this one